pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -327,6 +316,10 @@ func CreatePoller(
 				CallTick:  qemuAgentSysInterval,
 				InfoTypes: libvirt.DOMAIN_GUEST_INFO_LOAD,
 			},
+			{
+				CallTick:  qemuAgentFileInterval,
+				InfoTypes: libvirt.DOMAIN_GUEST_INFO_FILESYSTEM | libvirt.DOMAIN_GUEST_INFO_DISKS,
```

Comment:
Please put the second constant on a new line, so it looks similar to the above definition.